Makefile

```diff
@@ -704,6 +704,31 @@ clean-e2e-inst-signatures:
 #
 	$(CMD_RM) -rf $(OUTPUT_DIR)/e2e-inst-signatures
 
+
```

Comment:
This is a no for me, sorry. I love colors but dont want to start having to deal with ansi color codes and bugs born from that. Also, makes the makefile very ugly, hope you dont mind removing this.